.github/workflows/fpga.yml

```diff
@@ -331,6 +331,11 @@ jobs:
         run: |
           git submodule update --init dpe
 
+      - name: Install test dependencies
+        run: |
+          sudo apt install openocd
```

Comment:
FWIW, for this to work you would need to add -y here

Doesn't matter now that the FPGA image has these packages...